test/miss/map_test.exs

```diff
@@ -17,7 +17,7 @@ defmodule Miss.MapTest do
   end
 
   defmodule Metadata do
-    defstruct [:atom, :boolean, :decimal, :float, :integer]
+    defstruct [:atom, :boolean, :decimal, :float, :integer, map: %{}]
```

Comment:
Missing this map field in the doc:
- https://github.com/prodis/miss-elixir/pull/7/files#diff-aa979de4a7f1c3f93ef581cd3569fc42e0b0498566ec513a294589c087d13646R40-R42
- https://github.com/prodis/miss-elixir/pull/7/files#diff-aa979de4a7f1c3f93ef581cd3569fc42e0b0498566ec513a294589c087d13646R57-R58
- https://github.com/prodis/miss-elixir/pull/7/files#diff-aa979de4a7f1c3f93ef581cd3569fc42e0b0498566ec513a294589c087d13646R100-R101